cluster-scope/overlays/nerc-ocp-prod/feature/odf/kustomization.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+commonLabels:
+  nerc.mghpcc.org/feature: odf
+
+components:
+  - ../../../../components/nerc-secret-store
+
+resources:
+  - ../../../../bundles/odf-external
+  - externalsecrets/rook-ceph-external-cluster-details.yaml
```

Comment:
we should add the storageclass patch to this (it is commented out right now, so wouldn't matter).

Reply:
Good point, I've updated the pull request.